tests/unit/tests_high_five_base.cpp

```diff
@@ -1319,6 +1319,96 @@ BOOST_AUTO_TEST_CASE(HighFiveCompounds) {
 }
 
 
+enum Position {
+    FIRST = 1,
+    SECOND = 2,
+    THIRD = 3,
+    LAST = -1,
+};
+
+enum class Direction: signed char {
+    FORWARD = 1,
+    BACKWARD = -1,
+    LEFT = -2,
+    RIGHT = 2,
+};
+
+// This is only for boost test
+std::ostream& operator<<(std::ostream& ost, const Direction& dir) {
+    ost << static_cast<int>(dir);
+    return ost;
+}
+
+namespace HighFive {
+    EnumType<Position> create_enum_position() {
```

Comment:
We have an implicit constructor, no?
```suggestion
    EnumType<Position> create_enum_position() {
     return  {{"FIRST", Position::FIRST},
              {"SECOND", Position::SECOND},
              {"THIRD", Position::THIRD},
              {"LAST", Position::LAST}});
```